src/Talis/Persona/Client/Tokens.php

```diff
@@ -3,15 +3,12 @@
 use \Firebase\JWT\JWT;
 use \Firebase\JWT\ExpiredException;
 
-class ScopesNotDefinedException extends \Exception {
+class ScopesNotDefinedException extends \Exception
+{
 }
 
 class Tokens extends Base
 {
-    const VERIFIED_BY_JWT       =  'verified_by_jwt';
-    const VERIFIED_BY_PERSONA   =  'verified_by_persona';
-    const VERIFIED_BY_CACHE     =  'verified_by_cache';
```

Comment:
Not sure TBH. This release might need to be a breaking change. I hope that the consuming code wouldn't have cared how the token was verified.